app/src/main/java/timur/gilfanov/messenger/ui/screen/MainScreen.kt

```diff
@@ -0,0 +1,95 @@
+package timur.gilfanov.messenger.ui.screen
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Email
+import androidx.compose.material.icons.filled.Settings
+import androidx.compose.material3.Icon
+import androidx.compose.material3.NavigationBar
+import androidx.compose.material3.NavigationBarItem
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.navigation3.runtime.entry
+import androidx.navigation3.runtime.entryProvider
+import androidx.navigation3.runtime.rememberNavBackStack
+import androidx.navigation3.ui.NavDisplay
+import timur.gilfanov.messenger.navigation.Language
+import timur.gilfanov.messenger.navigation.ProfileEdit
+import timur.gilfanov.messenger.navigation.Settings
+import timur.gilfanov.messenger.ui.screen.user.LanguageScreen
+import timur.gilfanov.messenger.ui.screen.user.LanguageViewModel
+import timur.gilfanov.messenger.ui.screen.user.ProfileEditScreen
+import timur.gilfanov.messenger.ui.screen.user.ProfileEditViewModel
+import timur.gilfanov.messenger.ui.screen.user.UserScreen
+import timur.gilfanov.messenger.ui.screen.user.UserViewModel
+
+@Suppress("LongMethod") // remove suppression on implementation stage
+@Composable
+fun MainScreen(
+    chatNavDisplay: @Composable () -> Unit,
+    @Suppress("unused") modifier: Modifier = Modifier, // really not needed
```

Comment:
May be move lambda to the end to be trailing?
```suggestion
    @Suppress("unused") modifier: Modifier = Modifier, // really not needed
    chatNavDisplay: @Composable () -> Unit,    
```